src/utils/conversion.ts

```diff
@@ -2932,21 +2932,22 @@ export function transferRestrictionToPolymeshTransferCondition(
   ) {
     let rawMin;
     let rawMax;
+    const { min, max, claim, issuer } = value;
     if (type === TransferRestrictionType.ClaimCount) {
       restrictionType = 'ClaimCount';
-      rawMin = bigNumberToU64(value.min, context);
-      rawMax = optionize(bigNumberToU64)(value.max, context);
+      rawMin = bigNumberToU64(min, context);
+      rawMax = optionize(bigNumberToU64)(max, context);
     } else {
       // i.e. TransferRestrictionType.ClaimPercentage
       restrictionType = 'ClaimOwnership';
-      rawMin = percentageToPermill(value.min, context);
+      rawMin = percentageToPermill(min, context);
       rawMax = percentageToPermill(value.max, context);
```

Comment:
```suggestion
      rawMax = percentageToPermill(max, context);
```

Reply:
max got destructured earlier, so its type is `BigNumber | undefined` (optional for count).

I did this to satisfy the type checker without needing an assertion, or have another line for the assignment.

Reply:
Huh, thought the type checker was smarter. Fair enough